ui/src/app/components/apps-panel.spec.tsx

```diff
@@ -120,4 +134,134 @@ describe('AppsPanel', () => {
       expect(runtimeCost.text()).toContain(statusText);
     }
   );
+
+  // these tests assume that there are no Jupyter runtimes
+  // so what these tests actually show is whether Cromwell is an ActiveApp
+  test.each([
+    [AppStatus.RUNNING, true, true],
+    [AppStatus.STOPPED, true, true],
+    [AppStatus.STOPPING, true, true],
+    [AppStatus.STARTING, true, true],
+    [AppStatus.PROVISIONING, true, true],
+    [AppStatus.DELETING, true, true],
+    [AppStatus.STATUSUNSPECIFIED, true, true],
+    [AppStatus.ERROR, true, true],
+
+    // not visible [isVisible() = false]
+    [AppStatus.DELETED, false, true],
+
+    [null, false, true],
+  ])(
+    'should render / not render ActiveApps and AvailableApps when the Cromwell status is %s',
+    async (status, activeExpected, availableExpected) => {
+      runtimeStub.runtime.status = RuntimeStatus.Deleted;
+      appsStub.listAppsResponse = [{ status, appType: AppType.CROMWELL }];
+
+      const wrapper = await component();
+      expect(wrapper.exists()).toBeTruthy();
+      await waitOneTickAndUpdate(wrapper);
+
+      expect(findActiveApps(wrapper).exists()).toBe(activeExpected);
+      expect(findAvailableApps(wrapper, activeExpected).exists()).toBe(
+        availableExpected
+      );
```

Comment:
Could `availableExpected` be removed as a parameter? It's always `true` for these tests.

Reply:
good point!  I'll merge this as-is but I'll think about that on the branch I'm updating

Reply:
yeah this change looks good so it will be in a future PR